src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import './person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">I am {person.age}</p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Nested ternary looks complicated, let's move this part to a separate variable and use it here


`${person.sex === 'm' ? 'wife' : 'husband'`